spotify-conversion-next/services/spotify/spotifyService.ts

```diff
@@ -1,43 +1,39 @@
+import spotifyRoutes from "./spotifyRoutes";
+
 //Queries is local and service is server
+const querystring = require("node:querystring");
 
 export type SpotifyTokenResponse = {
   access_token: string;
   token_type: string;
-  expires_in: number;
+  expires_in: string;
   refresh_token: string;
   scope: string;
 };
 
 const myHeaders = new Headers({
-  Authorization: `Basic ${process.env.SPOTIFY_AUTH_TOKEN}`,
+  Authorization: `Basic ${Buffer.from(
+    process.env.SPOTIFY_ID + ":" + process.env.SPOTIFY_SECRET
+  ).toString("base64")}`,

```

Comment:
Make sure you have your `.env.local` file filled out.

Also, add one more value to it: ```NEXTAUTH_URL=http://localhost:3000```

that just tells next-auth what your website's base URL is.